src/clr-icons/shapes/media-shapes.ts

```diff
@@ -119,7 +127,11 @@ export const MediaShapes: any = {
     "headphones": ClrShapeHeadphones,
     "film-strip": ClrShapeFilmStrip,
     "music-note": ClrShapeMusicNote,
-    "wand": ClrShapeWand
+    "wand": ClrShapeWand,
```

Comment:
Why is the `wand` icon in media shapes?

Reply:
Yeah, that does seem odd. My original thinking was that it could be used for photo editing or something like that. It might make more sense to put it in Essentials - I'm OK with that.

Reply:
I have moved the shape into essential-set.